app/helpers/budgets_helper.rb

```diff
@@ -1,5 +1,9 @@
 module BudgetsHelper
 
+  def show_links_to_budget_investments(budget)
+    ['reviewing_ballots', 'finished'].include? budget.phase
```

Comment:
Maybe I wasn't explicity enough, but with ` current budget is on Balloting phase or later (reviewing_ballots or finished)` I actually meant balloting phase plus those two next phases as well.

Reply:
I'll fix the misunderstanding in a moment.